ejb3/src/main/java/org/jboss/as/ejb3/timerservice/persistence/database/DatabaseTimerPersistence.java

```diff
@@ -398,13 +398,14 @@ private void checkDatabase() {
                 try {
```

Comment:
the nested try block (line 398 and line 401) seems unnecessary. We can combine them into one.